_content/setup-wallets-commandline.md

```diff
@@ -0,0 +1,141 @@
+## Interledger Protocol (ILP) TestNet: Getting Started
+
+This tutorial describes how to use the command line and test XRP to:
+
+1. Create an account in the Xpring.io Interledger Testnet.
+2. Add test XRP to your account using the TestNet Rainmaker \(our version of a "faucet"\).
```

Comment:
> (our version of a "faucet"\)

I think should be in the 3rd person, since otherwise the perspective might assumed to be Xpring (which isn't preferable).